python/pyarrow/array.pxi

```diff
@@ -3153,7 +3153,7 @@ class FixedShapeTensorArray(ExtensionArray):
         Note: ``permutation`` should be trivial (``None`` or ``[0, 1, ..., len(shape)-1]``).
         """
         if self.type.permutation is None or self.type.permutation == list(range(len(self.type.shape))):
-            np_flat = np.asarray(self.storage.values)
+            np_flat = np.asarray(self.storage.flatten())
```

Comment:
I'm curious, is the underlying `Array` object guaranteed to be chunked and have a `flatten()` API? I tried digging through the code a bit, but had trouble finding the answer.

Reply:
The underlying type (storage type) of the `FixedShapeTensor` is a `FixedSizeList` which always has the `flatten()` API:
https://arrow.apache.org/docs/python/generated/pyarrow.FixedSizeListArray.html#pyarrow.FixedSizeListArray.flatten

Reply:
Awesome, thank you!